test/rtos_drivers/hil/src/individual_tests/i2c/rpc/master_write_multiple_test.c

```diff
@@ -64,16 +65,32 @@ static int main_test(i2c_test_ctx_t *ctx)
             }
             local_printf("MASTER sent %d", sent);
 
+            // Do another 1 byte transfer but with num_bytes_sent ptr = NULL
             ret = rtos_i2c_master_write(ctx->i2c_master_ctx,
                                         I2C_SLAVE_ADDR,
                                         (unsigned char*)&test_vector[i][1],
-                                        I2C_MASTER_WRITE_MULTIPLE_TEST_SIZE-1,
+                                        1,
+                                        NULL,
+                                        0);
+
+            if (ret != I2C_ACK)
+            {
+                local_printf("MASTER write failed to send 1");
+                return -1;
+            }
+            local_printf("MASTER sent %d", sent);
```

Comment:
Should you be using `sent` here because it wasn't a parameter to rtos_i2c_master_write since you are using NULL? Perhaps this should just be hard-coded as "MASTER sent 1".